code/index.html

```diff
@@ -1,30 +1,106 @@
+                    media="(max-width: 500px)"/>
+                <source srcset="images/dark.jpg" alt="deliscious" media="(max-width: 1200px)"/>
+                <img src="images/darkwide.jpg" alt="deliscious" class="heronormal"/>
+            </picture>
+        </header>
+
+        <div id="menu_button" class="menu">
+            <p class="menutxt">MENU</p>
+        </div>
+        <ul>
+            <li>
+                <a href="#">
+                    About</a>
+            </li>
+            <li>
+                <a href="#">
+                    FAQ</a>
+            </li>
+            <li>
+                <a href="form.html">
+                    FORM</a>
+            </li>
+        </ul>
+
+        <div class="main">
+            <div id="questions">
+                <h1 id="cookiequestions">Cookie Questions</h1>
+                <button class="dropacc">Do you have fat free cookies</button>
+                <div class="txt">
+                    <p>Yes we do, Lorem, ipsum dolor sit amet consectetur adipisicing elit. Officia
+                        atque suscipit ea voluptatem repellendus a ipsam consectetur culpa ullam?
+                        Magnam.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Molestiae
+                        atque, repellendus sequi dignissimos optio culpa!</p>
+                </div>
+
+                <button class="dropacc">Do you have gluten free cookies</button>
+                <div class="txt">
+                    <p>Yes we do, Lorem Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+                        Iure soluta molestias est reiciendis vel quasi deleniti, alias enim iste nulla
+                        totam illum placeat. Cupiditate laudantium cum nisi amet quos qui? ipsum dolor
+                        sit amet, consectetur adipisicing elit. Molestiae atque, repellendus sequi
+                        dignissimos optio culpa!</p>
+                </div>
+
+                <button class="dropacc">How many cookies can i order?</button>
+                <div class="txt">
+                    <p>Subscribe to our unlimited cookie delivery, Lorem ipsum dolor sit amet,
+                        consectetur adipisicing elit. Molestiae atque, repellendus sequi dignissimos
+                        optio culpa!</p>
+                </div>
+
+                <button class="dropacc">My cookies crumbled</button>
+                <div class="txt">
+                    <p>Impossible, Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quia,
+                        necessitatibus inventore! Dignissimos excepturi odio aliquam quidem nostrum
+                        repellat sunt eos, alias deserunt excepturi fugiat.dolor sit amet, consectetur
+                        adipisicing elit. Molestiae atque, repellendus sequi dignissimos optio culpa!</p>
+                </div>
+            </div>
+            <div class="linktoform">
+                <a href="form.html">
```

Comment:
This will give a nicer url! You don't need the `.html` part!
```suggestion
                <a href="/form">
```